### DISCUSSION TITLE
About the peripheral mouse

### DISCUSSION THREAD

**Discussion by Mi3sbbbi** [seerge/g-helper#1349] [General]
About peripherals Why is my peripherals not displayed on the IOS interface when I connect the mouse using Bluetooth? Previous peripherals appear to have appeared via a USB cable.

Is this normal in the current version, or is the bug just me? The device is Z13.

**Comment by seerge** [comment]
You will see supported mice only when they are connected via usb cable or a usb dongle

**Reply by Mi3sbbbi** [reply]
Is Bluetooth recognition included in the plan?

**Reply by seerge** [reply]
Nope :) BT protocol works differently to USB. And since you can already set (and save) your settings via USB it should be enough.

**Reply by Mi3sbbbi** [reply]
Ok, thank you very much for your answer. G-helper is such a great tool that I would never choose another brand of computer because of it. If one day G-helper can add Bluetooth recognition and screen color control mode to G-helper product features, I think it will be perfect. When it matures I think not only I should have a lot of people willing to pay for it. Oh maybe not until that day, if you need to unlock it now to use g-helper trust me I will buy it.
